services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -1180,13 +1210,13 @@ protected Answer registerTemplateOnSwift(DownloadCommand cmd) {
             try (FileInputStream fs = new FileInputStream(file)) {
                 md5sum = DigestUtils.md5Hex(fs);
             } catch (IOException e) {
-                s_logger.debug("Failed to get md5sum: " + file.getAbsoluteFile());
+                s_logger.error("Failed to get md5sum: " + file.getAbsoluteFile());
```

Comment:
Suggestion:

Pass the exception as parameter to `s_logger.error`:

```java
s_logger.error("Failed to get md5sum: " + file.getAbsoluteFile(), e);
```

Reply:
thanks will do it.